ARCs/arc-0033.md

```diff
@@ -0,0 +1,60 @@
+---
+arc: 33
+title: Becoming an xGov
+description: Explanation on how to become Expert Governors.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+xGov status gives a governor the right to vote on proposals raised by the community, to ratify the proposals as valid for final community vote, where said proposals pertain to spending a previously specified amount of Algo in a given epoch on particular initiatives.
+ 
+The purpose of this ARC is to define the standard by which xGov status is achieved.
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+### What is an xGov
+Expert Governors (xGov) are decentralized decision makers who demonstrate enduring commitment, are knowledgeable of the realities of the Algorand community, and whose interests are aligned with the good of blockchain.
+ 
+xGov **SHOULD** understand the community's needs and align their interests with those of the Algorand ecosystem.
+xGov **MAY** propose measures for a vote.
```

Comment:
Will there need to be a Quorum amongst XGov's before it goes out to the public for a vote or can an xGov propose an unlimited amount of things?

Reply:
Or it could be that they are limited to one per quarter

Reply:
I would also like clarification on whether xGovs are generation proposals are essentially sorting through community made proposals. Can an xGov make a proposal at all or is that a conflict of interest?

Reply:
> I would also like clarification on whether xGovs are generating these proposals or "designating" them from the community. Can xGov make a proposal at all or is that a conflict of interest?

Having a voting body be able to set its own agenda is a standard governance model used in many contexts.  I cannot fathom how that would be a conflict of interest in any way.

Reply:
> > I would also like clarification on whether xGovs are generating these proposals or "designating" them from the community. Can xGov make a proposal at all or is that a conflict of interest?
> 
> Having a voting body be able to set its own agenda is a standard governance model used in many contexts. I cannot fathom how that would be a conflict of interest in any way.

What I mean is that is not clearly written out that xGovs are allowed to participate in the act of proposal generation. I would assume the case, but I'm not sure from the wording. As to how there could be a conflict of interest, I could think of a few reasons why a particular xGov candidate that may stand to profit directly from a proposal in xGov hands might need to abstain from a vote. It is also possible, depending on how xGovs might be selected for a particular vote, that might unknowingly place themself in an unethical position. This would all depend on exactly how this was structured, in my mind.

Reply:
1. The xGov group may overlap significantly with the engaged core of the community, and excluding them from writing proposals would exclude many of the very people whom you would want to be crafting policies and proposals.  That seems antithetical to what we're trying to achieve here.
2. Any proposal written by an xGov, even if entirely self-serving, would need to be approved by a majority of xGovs to be made into a full governance measure and then passed by a majority in a full governance vote.  This is a significant hurdle to clear for someone to benefit from a self-serving proposal.  For this reason, I don't think it would be necessary to take away xGov's ability to write proposals.